### PR TITLE
docs: fix typo and improve domain page

### LINKED ISSUE #4406: Documentation typo
> To set up an application, you can just run copilot init. You'll be asked if you want to set up an app or choose an use app.

I believe this should be, "... you want to set up an app or choose an existing app."

### PR DESCRIPTION
**Pull request by efekarakus**
Fixes #4406

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/developing/domain.en.md [hunk 1]
```diff
@@ -70,16 +73,25 @@ Under the hood, Copilot
...
 * creates an optional A record for your alias
 
-**What does it look like?**
+#### What does it look like?
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/Oyr-n59mVjI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ### Use domain in your existing validated certificates
-If you own a domain outside of Route53 or for compliance reasons want to use an `alias` from your existing ACM certificates, you can specify the `--import-cert-arns` flag when creating the environment to import validated ACM certificates that include the alias. For example:
+If you'd like more granular control over the generated ACM certificate or the [default `alias` options](#customized-domain-alias) aren't flexible enough, you can specify the `--import-cert-arns` flag when creating the environment to import validated ACM certificates that include the alias. For example:
```
Comment (huanjani) [inline site/content/docs/developing/domain.en.md]:
```suggestion
If you'd like more granular control over the generated ACM certificate, or if the [default `alias` options](#customized-domain-alias) aren't flexible enough, you can specify the `--import-cert-arns` flag when creating the environment to import validated ACM certificates that include the alias. For example:
```

File: site/content/docs/developing/domain.en.md [hunk 2]
```diff
@@ -91,7 +103,7 @@ http:
   alias: example.aws
   hosted_zone: Z0873220N255IR3MTNR4
 ```
-2. Deploy the service without the `hosted_zone` field, then manually add the DNS name of the Application Load Balancer (ALB) created in that environment as an A record where your alias domain is hosted.
+2. Alternatively, deploy the service without the `hosted_zone` field, then manually add the DNS name of the Application Load Balancer (ALB) created in that environment as an A record where your alias domain is hosted.
```
Comment (huanjani) [inline site/content/docs/developing/domain.en.md]: Is this redundant with the "you can either" above? Maybe drop "either" and take out the numbering? Not sure what makes the most sense here.

Comment (efekarakus) [reply site/content/docs/developing/domain.en.md]: > Maybe drop "either" and take out the numbering? 

Done!

**Review by iamhopaul123 — APPROVED** [review]
🚢

**Review by huanjani — APPROVED** [review]
:shipit:

**Pull request merged**